configuration.nix

```diff
@@ -42,7 +40,9 @@ in {
 
   programs.bash.enableCompletion = true;
 
-  users.users = {
+  users.users = let
+    posh = pkgs.posh { image = "quay.io/enarx/fedora"; global_args = "--device /dev/sgx/enclave"; };
```

Comment:
One downside to this is that I lose the ability to gate access to SGX on a per-user basis. Not the end of the world, but something.

Reply:
You can do override this on reference like so: `posh.overrideAttrs(attrs: attrs // { global_args = "" })`

Reply:
Okay, WFM.